symplyphysics/laws/electricity/circuits/total_conductivities_matrix.py

```diff
@@ -0,0 +1,57 @@
+from sympy import Eq, solve, Matrix
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+## Description
+## The conductivity matrix is one of the ways to describe a microwave device. The Y-parameters of the device act as elements
+## of this matrix. The matrix equation relates the input and output currents to the input and output voltages.
+
+## Law is: [I1, I2] = Matrix([[Y11, Y12], [Y21, Y22]]) * [U1, U2], where
+## U1 - input voltage,
+## U2 - output voltage,
+## Y11 - ratio of input current to input voltage in case of a short circuit at the output,
+## Y12 - ratio of input current to output voltage in case of a short circuit at the output,
+## Y21 - ratio of output current to input voltage in case of a short circuit at the output,
+## Y22 - ratio of output current to output voltage in case of a short circuit at the output,
+## I1 - input current,
+## I2 - output current.
+
+input_voltage = Symbol("input_voltage", units.voltage)
+output_voltage = Symbol("output_voltage", units.voltage)
+input_current = Symbol("input_current", units.current)
+output_current = Symbol("output_current", units.current)
+conductivity_11 = Symbol("conductivity_11", units.conductance)
+conductivity_12 = Symbol("conductivity_12", units.conductance)
+conductivity_21 = Symbol("conductivity_21", units.conductance)
+conductivity_22 = Symbol("conductivity_22", units.conductance)
+
+law = Eq(Matrix([input_current, output_current]), Matrix([[conductivity_11, conductivity_12], [conductivity_21, conductivity_22]]) * Matrix([input_voltage, output_voltage]))
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(input_current_=input_current, output_current_=output_current,) #conductivities_=conductivity_11)
```

Comment:
Where is conductivity validation?

Reply:
> Where is conductivity validation?

I commented out this part of the code because it has the same problem that I wrote about in the previous task:
https://github.com/blackyblack/symplyphysics/pull/579#issue-2218623106

I can't set the "conductivities_" for "@validate_input" correctly. Because of this, checking the values of the impedance matrix fails.

Reply:
Yes, you cannot do it because you have tuple of tuples. You should use `assert_equivalent_dimension` to manually check every input for desired dimension.